53/bryankimani/artist_albums_names.py

```diff
@@ -0,0 +1,61 @@
+import os
+import spotipy
+import spotipy.oauth2 as oauth2
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def print_header():
+    print("-------------------------------------")
+    print("         ARTIST ALBUMS NAMES         ")
+    print("-------------------------------------")
+    print()
+
+
+def spotify_auth():
+    client_credentials_manager = oauth2.SpotifyClientCredentials(
+        client_id=os.getenv("CLIENT_ID"), client_secret=os.getenv("CLIENT_SECRET")
+    )
+    sp = spotipy.Spotify(client_credentials_manager=client_credentials_manager)
+    return sp
+
+
+def get_artist_id():
+    # return the artist id from the entered artist name
+    artist_name = input("Please Enter Artist Name: ")
+    results_artist_name = spotify_auth().search(
```

Comment:
I would pull the auth out doing it once, then use the sp variable rather

Reply:
Great. Thanks for the pointer.